src/internal/agent/start.go

```diff
@@ -27,35 +28,40 @@ const (
 )
 
 // StartWebhook launches the Zarf agent mutating webhook in the cluster.
-func StartWebhook() {
+func StartWebhook(ctx context.Context) error {
 	message.Debug("agent.StartWebhook()")
-
-	startServer(agentHttp.NewAdmissionServer(httpPort))
+	return startServer(ctx, agentHttp.NewAdmissionServer(httpPort))
```

Comment:
This context should be used in the agent, but this is fine to merge so I'll just spin up another PR to do that